search/.eslintrc.yml

```diff
@@ -6,6 +6,10 @@ extends:
   - eslint:recommended
 parserOptions:
   ecmaVersion: 12
+  sourceType: module
 rules:
   no-console: off
   radix: off
+  semi: 2
+  no-trailing-spaces: 2
+  block-spacing: 2
```

Comment:
Value should be "always" or "never"

Reply:
Also switched to "error"

Reply:
https://eslint.org/docs/user-guide/configuring#configuring-rules